MODULE.bazel

```diff
@@ -6,7 +6,7 @@ module(
     compatibility_level = 1,
 )
 
-bazel_dep(name = "aspect_bazel_lib", version = "1.31.2")
+bazel_dep(name = "aspect_bazel_lib", version = "1.32.0")
```

Comment:
this wasn't synced with workspace one somehow 🤷‍♂️

Reply:
I believe this is intentional since the MODULE.bazel can be minimal version (since bzlmod will resolve the latest from all requested versions from all dependencies) but the WORKSPACE should be latest because of how confusing transitives are in WORKSPACES

Reply:
now that you're using expand_template from bazel-lib would be the right time to bump the minimum tho :)

Reply:
right. I was about to say that. :P